app/controllers/application_controller.rb

```diff
@@ -2,11 +2,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  before_filter :get_next_meetup
+  before_filter :get_next_meetup, :get_events
 
   def get_next_meetup
     @next_meetup = Meetup.next
   end
 
-
+  def get_events
+    @events = Event.published
```

Comment:
It's actually used in the [sidebar](https://github.com/montrealrb/Montreal.rb/blob/master/app/views/layouts/_sidebar.html.erb#L2-L17) as well to display upcoming events. Since the sidebar is part of the application layout, I moved it to the `application_controller`